src/plugins/Show.tsx

```diff
@@ -0,0 +1,24 @@
+import * as _ from "lodash";
+import * as React from "react";
+import {homeDirectory, pluralize, resolveDirectory, resolveFile, mapObject} from "../utils/Common";
+import {existsSync, statSync} from "fs";
+import {PluginManager} from "../PluginManager";
+import {Job} from "../shell/Job";
+
+PluginManager.registerOutputDecorator({
+    decorate: (job: Job): React.ReactElement<any> => {
+        const rows = job.screenBuffer.toLines().map(row => {
```

Comment:
Had you resolved paths in the command itself this could be just `const rows = job.screenBuffer.toLines().map(path => <img src={path}/>)`